### PR TITLE
feat: add net.natesales.Aviator

### PR DESCRIPTION
**Pull request by natesales**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
Host filesystem access for media selection.
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: net.natesales.Aviator.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    sources:
+      - type: archive
+        url: https://ffmpeg.org/releases/ffmpeg-5.1.tar.xz
+        sha256: 55eb6aab5ee235550fa54a33eaf8bf1b4ec66c01453182b12f6a993d75698b03
+
+  - name: aviator
+    builddir: true
+    buildsystem: meson
+    sources:
+      - type: dir
```
Comment (barthalion) [inline net.natesales.Aviator.yml:60]: This needs to point to some external git repository.

Comment (natesales) [reply net.natesales.Aviator.yml]: Oops, fixed!

File: net.natesales.Aviator.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: "42"
+
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=host
```
[the remark this answers is not in the document]
Comment (natesales) [reply net.natesales.Aviator.yml:12]: Flatpak portals are planned for the next release. It's currently using this for media file selection with a simple file picker.

File: net.natesales.Aviator.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hfiguiere) [inline net.natesales.Aviator.metainfo.xml:1]: this file is upstream, why do you need it here. it's not even used.

File: net.natesales.Aviator.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (hfiguiere) [reply net.natesales.Aviator.yml:12]: still very broad.

[the remark this answers is not in the document]
Comment (hfiguiere) [reply net.natesales.Aviator.yml:12]: `xdg-videos` would be a good choice.

**Comment by hfiguiere** [comment]
bot, build net.natesales.Aviator

**Pull request closed without merging**